src/ontology/mondo-ingest.Makefile

```diff
@@ -598,6 +598,22 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+
+#################
+##### Ad hoc ####
+#################
+tmp/ORDO_en_4.4.owl:
+	wget https://www.orphadata.com/data/ontologies/ordo/last_version/ORDO_en_4.4.owl -O $@
```

Comment:
## `component-download` due for an update?
- [ ] @matentzn respond

Should I also update `mondo-ingest-odk.yaml` to use this URL instead?

Reply:
Look at 

```
$(COMPONENTSDIR)/ordo.owl: $(TMPDIR)/ordo_relevant_signature.txt config/properties.txt | component-download-ordo.owl
	if [ $(COMP) = true ]; then $(ROBOT) remove -i $(TMPDIR)/component-download-ordo.owl.owl --select imports \
		merge ...
```

It is better to:

1. depend in `component-download-ordo.owl`
2. And use `$(TMPDIR)/component-download-ordo.owl.owl`

to ensure we are using the same version as the rest of the pipeline

Reply:
I moved this comment to a [new ticket](https://github.com/monarch-initiative/mondo-ingest/issues/511) since it's work not directly related to this PR, despite being informed by this PR